seq2science/rules/quantification.smk

```diff
@@ -166,22 +166,45 @@ elif config["quantifier"] == "kallistobus":
             {params.options} > {log} 2>&1
             """
 
-    def get_kallistobus_reads(wildcards):
-        reads = []
-        sample = wildcards.sample
+    def get_fastq_pair_reads(wildcards):
+        reads = dict()
         assert sampledict[sample]["layout"] == "PAIRED"
```

Comment:
nitpick (its not even in this PR, sorry): add an error message if the assert is tripped. Something like
`assert sampledict[sample]["layout"] == "PAIRED", "we only support paired-end scRNA-seq"`

Reply:
Added error message